compiler/modules/front/token.bal

```diff
@@ -80,6 +80,10 @@ final readonly & map<MultiCharDelim> WITH_EQUALS = {
     ">": ">="
 };
 
+const MODE_NORMAL = 1;
+const MODE_TYPE = 2;
```

Comment:
I think `MODE_TYPE_DESC` would be clearer than `MODE_TYPE`.

I would make `MODE_NORMAL` be 0, rather than 1.